build-scripts/hack/util.py

```diff
@@ -48,3 +49,9 @@ def read_file(path: Path) -> str:
 
 def read_url(url: str) -> str:
     return urlopen(url).read().decode().strip()
+
+def helm_pull(chart_name: str, repo_url: str, version: str, destination: Path) -> None:
+    parse_output(["helm", "repo", "add", chart_name, repo_url])
+    parse_output(["helm", "pull", f"{chart_name}/{chart_name}", "--version", version, "--destination", destination])
```

Comment:
please stay aware for conflicts with https://github.com/canonical/k8s-snap/pull/468/files/e52a1b2ce1fd4207e9772a40d86b018d8f93abb8#diff-c6cdac9381e357f73ccfe7a568a0468768bb5eadb2f32c4ba86db4d05d722c4fR52

also, please use `helm pull --repo $repo ...` instead of changing the repositories of the local helm client, as this is a recipe for unintended side-effects